### DISCUSSION TITLE
国密workshop讨论

### DISCUSSION THREAD

**Discussion by SamYuan1990** [Hyperledger-TWGC/fabric-gm-wiki#23] [Show and tell]
- 提议人 龙文选： 国密的workshop/meet up 
- 时间：8月中 
- 目的/目标：国密项目推广 
- 效果：可以用，如何用，使用中遇到问题的解答 超级账本/信通院 重点讲一个 
- 形式 讲透 开篇 以某个国密基础库为例，做demo：

响应： Sam
我们可不可以在workshop中写一个server/client的demo
包括
- 做sm3+sm2 摘要+签名/验证 操作
- 加密解密sm2的操作
- sm4？加密解密操作
- 双证书国密通讯？(TBD)

**Comment by SamYuan1990** [comment]
听众门槛，workshop组织难度，场地，收益。

**Reply by davidkhala** [reply]
听众门槛：提高现场的听众门槛（设备OS，go语言的问题），比如10个人特别有兴趣的，是否有一个切实的需求

**Reply by davidkhala** [reply]
主讲人[待定]：陈序

**Reply by davidkhala** [reply]
时间：8月份的周末

**Reply by davidkhala** [reply]
场地要求：可用的wifi，接线板，录屏，zoom/VooV

**Comment by zxm256** [comment]
重点是围绕基础库的使用来介绍么？

**Reply by SamYuan1990** [reply]
是的。目前的想法聚焦于：可以用，如何用，使用中遇到问题的解答 。 从而达到国密项目推广的目的/目标。

**Reply by SamYuan1990** [reply]
赵博这边有什么其他建议和观点？

**Comment by SamYuan1990** [comment]
表单？demo repo？ppt？录屏？直播？ 形式有待明确。
